tests/integration_tests/test_integration_postgres_example.py

```diff
@@ -26,6 +26,7 @@ def postgres_example_db() -> Generator:
     engine.dispose()
 
 
+@pytest.mark.integration_postgres
```

Comment:
Similar test in `test_integration_mongodb_example.py` could be marked with mongo - and test_integratino_mssql_example.py marked with mssql